### PR TITLE
UI form update and made accessible

### LINKED ISSUE #26: Verify the UI is accessible

### PR DESCRIPTION
**Pull request by emma-hubka**
Closes #26 Verify the UI is accessible

### CODE REVIEW

File: extension/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <input type="checkbox" id="blind" name="blind" value="Blind" />
+            Blind</label
+          ><br />
+
+          <label for="deaf">
+            <input type="checkbox" id="deaf" name="deaf" value="Deaf" />
+            Deaf</label
+          ><br />
+
+          <label for="blind"
```
Comment (Bracciata) [inline extension/index.html]: This should be colorBlind

File: extension/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            ><input
+              type="checkbox"
+              id="colorblind"
+              name="colorblind"
+              value="Colorblind"
+            />
+            Blind</label
```
Comment (Bracciata) [inline extension/index.html]: Update this

File: extension/index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            <input type="checkbox" id="deaf" name="deaf" value="Deaf" />
+            Deaf</label
+          ><br />
+
+          <label for="blind"
+            ><input
+              type="checkbox"
+              id="colorblind"
+              name="colorblind"
+              value="Colorblind"
```
Comment (Bracciata) [inline extension/index.html]: value should not be Colorblind for a checkbox. It should always be true or false.

File: extension/index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      </div>
+
+      <div class="sub-title">What applies to you?</div>
+
+      <div class="description">
+        <form>
+          <label for="blind">
+            <input type="checkbox" id="blind" name="blind" value="Blind" />
+            Blind</label
+          ><br />
```
Comment (Bracciata) [inline extension/index.html:40]: Odd formatting.

**Review by Bracciata — CHANGES_REQUESTED** [review]
Overall great work but some work is needed regarding the form in particular.

File: extension/index.html (hunk 1 above)
Comment (emma-hubka) [reply extension/index.html]: ✨

**Comment by Bracciata** [comment]
Also noticed horizontal scrolling which needs to be fixed.

File: extension/index.html (hunk 4 above)
Comment (emma-hubka) [reply extension/index.html:40]: that's how my auto formatter formatted it

**Pull request merged**